deploy-board/deploy_board/templates/groups/group_config.tmpl

```diff
@@ -152,6 +152,36 @@
                         <span class="input-group-addon">Minutes</span>
                         </div>
                     </div>
+                    <label for="lifecycle_notifications" class="deployToolTip control-label col-xs-2"
+                        data-toggle="tooltip"
+                        title="Receive email notifications for lifecycle events">
+                        Lifecycle Notifications
```

Comment:
Instead of calling Lifecycle Notfication, can you call it "Instance Launch/Termination Notification" ?